cli/viam/main.go

```diff
@@ -88,21 +87,26 @@ func main() {
 						return err
 					}
 
-					loggedInMessage := func(token *rdkcli.Token) {
-						fmt.Fprintf(c.App.Writer, "Already logged in as %q, expires %s\n", token.User.Email,
+					loggedInMessage := func(token *rdkcli.Token, alreadyLoggedIn bool) {
+						var already string
+						if alreadyLoggedIn {
+							already = "already "
```

Comment:
We were printing `Already logged in` when a user had just logged in for the first time.